pkg/config/kubeClient.go

```diff
@@ -32,7 +33,7 @@ func GetDynamicClient() dynamic.Interface {
 	return dynamicClient
 }
 
-func getKubeConfig() *rest.Config {
+func GetKubeConfig() *rest.Config {
```

Comment:
Is this change needed? I don't see `GetKubeConfig()` being used from another package.

Reply:
Updated.